src/dataExplorer/components/ScriptQueryBuilder.tsx

```diff
@@ -105,6 +110,9 @@ const ScriptQueryBuilder: FC = () => {
     }
   }, [handleClear, hasChanged])
 
+  const filterOutFluxInIOx = option =>
```

Comment:
To hammer on this point some more, this _is_ an action/verb where `filterOutFluxInIOx` is a method, rather than a boolean value. Without any context into what's happening, the variable name can give a clue to what the intent of this code is